install/kubernetes/Makefile

```diff
@@ -1,29 +1,25 @@
 # Copyright Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-include Makefile.digests ../../Makefile.defs
+MAKEFILE_VALUES?=Makefile.values
+include $(MAKEFILE_VALUES)
 
-HUBBLE_UI_VERSION := "v0.9.0@sha256:0ef04e9a29212925da6bdfd0ba5b581765e41a01f1cc30563cef9b30b457fea0"
-HUBBLE_UI_BACKEND_VERSION := "v0.9.0@sha256:000df6b76719f607a9edefb9af94dfd1811a6f1b6a8a9c537cba90bf12df474b"
-MANAGED_ETCD_VERSION := "v2.0.7"
-ETCD_VERSION := "v3.5.2@sha256:70fe70b6cdaea99fa9e99d3a16483954eb084c3b32a2475abea4e709a793636c"
-NODEINIT_VERSION := "d69851597ea019af980891a4628fb36b7880ec26"
-CERTGEN_VERSION := "v0.1.8"
+include ../../Makefile.defs
+
+ifeq ($(CILIUM_VERSION),)
+export CILIUM_VERSION:="v$(VERSION)"
+endif
 
 MIN_K8S_MAJOR := 1
 MIN_K8S_MINOR := 16
 MIN_K8S_VERSION := "v$(MIN_K8S_MAJOR).$(MIN_K8S_MINOR).0"
 
-CILIUM_CHARTS := "$(ROOT_DIR)/$(RELATIVE_DIR)/cilium"
+CILIUM_CHARTS := $(ROOT_DIR)/$(RELATIVE_DIR)/cilium
 CILIUM_VALUES := "$(CILIUM_CHARTS)/values.yaml"
```

Comment:
`CILIUM_VALUES` seems unused after this patch. It is tempting to define something like:

```makefile
CILIUM_VALUES_TEMPLATE := $(CILIUM_VALUES).tmpl
```
And use both variable, but I would lean towards removing `CILIUM_VALUES` instead because the values files are only used by this Makefile (i.e. the relative path is enough) and the former solution make `git grep values.yaml.tmpl` worse.